mti_nma/steps/singlecell/singlecell.py

```diff
@@ -55,7 +58,8 @@ def run(self, cell_line_id='AICS-13', nsamples=16, **kwargs):
 
             :: IMPORTANT:
             The line
-                raw = AICSImage(df.ReadPathRaw[FOVId]).get_image_data('ZYX', S=0, T=0, C=-2)
+                raw = AICSImage(
+                    df.ReadPathRaw[FOVId]).get_image_data('ZYX', S=0, T=0, C=-2)
             may not return the nuclear channel for cell lines other than Lamin.
```

Comment:
Do you know if there is metadata for these files? You may be able to use the metadata to look for the channel names. Like try for channel names and if they aren't there then default to `C=-2` or similar?

Reply:
@vianamp what do you think?

Reply:
[AICSImage objects have a `get_channel_names` function so maybe it will work?](https://allencellmodeling.github.io/aicsimageio/aicsimageio.html#aicsimageio.aics_image.AICSImage.get_channel_names)

Reply:
Using that function returns:
`['CMDRP_1', 'CMDRP', 'EGFP_1', 'EGFP', 'H3342_1', 'H3342', 'Bright_100X']`
And we were using C=-2. Is H3342 the nuclear channel? If so then I will replace this with:
`ch_ind = AICSImage(
                    df.ReadPathRaw[fov_id]).get_channel_names().index('H3342')
                raw = AICSImage(
                    df.ReadPathRaw[fov_id]).get_image_data("ZYX", S=0, T=0, C=ch_ind)`

Reply:
That is a question for microscopy I think? I used to remember if the correct channel was `H3342_1` or `H3342` and I even had a function to pick the correct one depending on which channel names were there but I don't remember anymore.

I _think_ its: if `.._1` is present then use that one, otherwise use `H3342` base. But again, check with microscopy.

Reply:
Thanks! checked with Matheus who has talked to Calysta about it and the _ channel is a dummy. Udpated the code to resolve this.